Add webp to array of mimetypes
===================================================================
--- a/File/Mime.php
+++ b/File/Mime.php	(date 1717557642000)
@@ -43,6 +43,7 @@
         'jpe' => 'image/jpeg',
         'jpeg' => 'image/jpeg',
         'jpg' => 'image/jpeg',
+        'webp' => 'image/webp',
         'gif' => 'image/gif',
         'bmp' => 'image/bmp',
         'ico' => 'image/vnd.microsoft.icon',
