Add WebP to array of mimetypes
===================================================================
--- a/Model/File/Validator/Image.php
+++ b/Model/File/Validator/Image.php	(date 1737127347006)
@@ -25,6 +25,7 @@
         'jpe'  => 'image/jpeg',
         'jpeg' => 'image/jpeg',
         'jpg'  => 'image/jpeg',
+        'webp' => 'image/webp',
         'gif'  => 'image/gif',
         'bmp'  => 'image/bmp',
         'ico'  => ['image/vnd.microsoft.icon', 'image/x-icon']
