Add WebP to array of extensions
===================================================================
--- a/Helper/Storage.php
+++ b/Helper/Storage.php	(date 1737129649620)
@@ -337,7 +337,7 @@
     {
         return $this->getStorageType() == \Magento\Theme\Model\Wysiwyg\Storage::TYPE_FONT
             ? ['ttf', 'otf', 'eot', 'svg', 'woff']
-            : ['jpg', 'jpeg', 'gif', 'png', 'xbm', 'wbmp'];
+            : ['jpg', 'jpeg', 'gif', 'png', 'xbm', 'wbmp', 'webp'];
     }

     /**
